sprout/scripts/config_validation/scscf_schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "title": "All SCSCF servers",
+  "type": "object",
+  "properties": {
+    "s-cscfs": {
+      "type": "array",
+      "items": {
+        "title": "Each SCSCF server",
+        "type": "object",
+        "properties": {
+          "server": {
```

Comment:
I only put in where there is apparent need for clarification; expected most properties to be self-explanatory